client/my-sites/stats/stats-insights/index.jsx

```diff
@@ -35,6 +36,11 @@ const StatsInsights = ( props ) => {
 	// Necessary to properly configure the fixed navigation headers.
 	sessionStorage.setItem( 'jp-stats-last-tab', 'insights' );
 
+	const lastYear = moment.utc().subtract( 1, 'year' ).format( 'YYYY-MM-DD' );
+	const isPrevArrowHidden = ( queryDate ) => {
+		return moment( queryDate ).isSameOrBefore( '2001-01-01' );
```

Comment:
Could we avoid using `moment` here and directly use the built-in Date object? We should be able to use the [`getFullYear()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getFullYear) function.